Mathlib/Tactic/Linter/UpstreamableDecl.lean

```diff
@@ -21,8 +21,12 @@ def Lean.Name.isLocal (env : Environment) (decl : Name) : Bool :=
 
 open Mathlib.Command.MinImports
 
-/-- Does the declaration with this name depend on `def`s in the current file? -/
-def Lean.Environment.localDefDependencies (env : Environment) (stx id : Syntax) :
+/-- Does the declaration with this name depend on definitions in the current file?
+
+Here, "definition" means everything that is not a theorem, and so includes `theorem`,
```

Comment:
```suggestion
Here, "definition" means everything that is not a theorem, and so includes `def`,
```
I guess!  :smile:

Reply:
You are absolutely right of course, well spotted :D